pkgs/development/libraries/libint/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib, stdenv, fetchFromGitHub, autoconf, automake, libtool
+, python3, perl, gmpxx, mpfr, boost, eigen, gfortran
+, enableFMA ? false
+} :
+
+stdenv.mkDerivation rec {
+  pname = "libint2";
+  version = "2.6.0";
+
+  src = fetchFromGitHub {
+    owner = "evaleev";
+    repo = "libint";
+    rev = "v${version}";
+    sha256 = "0pbc2j928jyffhdp4x5bkw68mqmx610qqhnb223vdzr0n2yj5y19";
+  };
+
+  patches = [
+    ./fix-paths.patch
+  ];
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    libtool
+    gfortran
+    mpfr
+    python3
+    perl
+    gmpxx
+  ];
+
+  buildInputs = [ boost ];
+
+  enableParallelBuilding = true;
+
+  doCheck = true;
+
+  configureFlags = [
+    "--enable-eri=2"
+    "--enable-eri3=2"
+    "--enable-eri2=2"
+    "--with-eri-max-am=7,5,4"
+    "--with-eri-opt-am=3"
+    "--with-eri3-max-am=7"
+    "--with-eri2-max-am=7"
+    "--with-g12-max-am=5"
+    "--with-g12-opt-am=3"
+    "--with-g12dkh-max-am=5"
+    "--with-g12dkh-opt-am=3"
+    "--enable-contracted-ints"
+    "--enable-shared"
+   ] ++ lib.optional enableFMA "--enable-fma";
+
+  preConfigure = ''
+    ./autogen.sh
+  '';
+
+  postBuild = ''
+    # build the fortran interface file
+    cd export/fortran
+    make libint_f.o ENABLE_FORTRAN=yes
+    cd ../..
+  '';
+
+  postInstall = ''
+    cp export/fortran/libint_f.mod $out/include/
+  '';
+
+  meta = with lib; {
+    description = "Library for the evaluation of molecular integrals of many-body operators over Gaussian functions";
+    homepage = "https://github.com/evaleev/libint";
+    license = with licenses; [ lgpl3Only gpl3Only ];
+    maintainers = [ maintainers.markuskowa ];
+    platforms = platforms.linux;
+    timeout = 3 * 3600; # Long build time, not parallelized.
```

Comment:
Can we only set this when hydra really times out or are you absolutely sure about this?

Reply:
The test build of this PR took a little bit more than two hours on an EPYC Rome system. It could even be that 3 hours is a bit too tight for Hydra. Unfortunately neither the code generation nor the actual compilation of the generated code runs on more than one CPU, making this a notorious long-runner build.

Reply:
Is there a sensible _subset_ of the tests that could be selected for running?

Reply:
This is pure build time: A code generator script creates C++ code (~1hour) and then it is compiled (>1hour).

Reply:
```suggestion
```

We really only need to set this if we know that the tests are flaky. Otherwise, you're just cancelling valid builds on loaded machines, wasting maintainer time and build time when restarted.

Reply:
@roberth Just to clarify: the long build time has nothing to with tests. It is simply an overly long serial process of code generation followed by its compilation. IIRC the standard timeout is 1h on hydra? If that is the case the build will definitely fail with the default timeout.

Reply:
(wrong robert ;) Understood. Doesn't sound like there's an easy workaround.

Reply:
The default timeout is 10h total or 2h of log silence.

Reply:
I didn't know that but then it should work without the timeout.